featurebyte/api/static_source_table.py

```diff
@@ -54,6 +54,18 @@ def to_pandas(self) -> pd.DataFrame:
         """
         return super().to_pandas()
 
+    def to_spark_df(self) -> Any:
+        """
+        Get a spark dataframe from the table.
+
+        Returns
+        -------
+        Any
+            Spark DataFrame
+        """
+
+        return super().to_spark_df()
```

Comment:
Why is this method overridden without any change? Is it to customize the docstring?

Reply:
you are right - I forgot to update the docstrings